torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -638,7 +638,7 @@ def __call__(
 # Used for wrapping a Triton Kernel in a functional manner
 class TritonKernelWrapperFunctional(HigherOrderOperator):
     def __init__(self) -> None:
-        super().__init__("triton_kernel_wrapper_functional", cacheable=False)
+        super().__init__("triton_kernel_wrapper_functional", cacheable=True)
```

Comment:
I'm confused, FXGraphCache should only see one of {triton_kernel_wrapper_mutation, triton_kernel_wrapper_functional} (depending on where the cache is in Inductor), did we need to make both cacheable?

Reply:
No harm, the problem is mostly that the test coverage only covers one of them so if we didn't actually implement the other one correctly and it somehow shows up in in the future then problems